pkg/jobrunaggregator/jobrunbigqueryloader/cmd.go

```diff
@@ -29,6 +32,8 @@ func NewBigQueryTestRunUploadFlags() *BigQueryTestRunUploadFlags {
 func (f *BigQueryTestRunUploadFlags) BindFlags(fs *pflag.FlagSet) {
 	f.DataCoordinates.BindFlags(fs)
 	f.Authentication.BindFlags(fs)
+
+	fs.BoolVar(&f.DryRun, "dry-run", f.DryRun, "Run the command, but don't mutate data.")
```

Comment:
I added this so that I could have confidence in running this command locally to test.